<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>舞蹈比赛小助手</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="MatchgroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>比赛进行中：</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="MgroupBoxPro">
         <property name="title">
          <string>题目</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="MlabelPro">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>197</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="MButtonGrade">
            <property name="text">
             <string>开始打分</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="MarchButtonDanceSin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>1.独舞</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="MarchButtonPro">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>2.文化提问</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="MarchButtonDanceOrd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>3.命题舞蹈</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="Mlabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>当前参赛者：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MlabelPeopel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QGroupBox" name="LookPeoplegroupBox">
      <property name="title">
       <string>选手查看台</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="LookPeoplecomboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>姓名:

年龄:

性别:

地址:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="LooKPeolplelabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="LookPeopleButtonGrade">
         <property name="text">
          <string>查看成绩</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>220</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonAgain">
      <property name="text">
       <string>再赛一场</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonControl">
      <property name="text">
       <string>新的比赛</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
